### PR TITLE
Revert "Revert PR #56 through #76 (#104)"

### PR DESCRIPTION
**Pull request by VMSolidus**
This reverts commit 648fea35b6a1f080a0d02577ff610e5c17c7a7b3.

# Description
Now that we have the list of Wizden Mirrors, it's now possible to bring this back, and resume work on maintaining this repo.

### CODE REVIEW

**Review by DangerRevolution — APPROVED** [review]
not sure why this was reverted but cool changes :shrug:

**Comment by LovelyLophi** [comment]
big fan of lunchboxes

**Comment by DangerRevolution** [comment]
> big fan of lunchboxes

<3

File: Content.Server/Ame/AmeNodeGroup.cs [hunk 1]
```diff
@@ -182,7 +182,7 @@ public float CalculatePower(int fuel, int cores)
         // Fuel is squared so more fuel vastly increases power and efficiency
         // We divide by the number of cores so a larger AME is less efficient at the same fuel settings
         // this results in all AMEs having the same efficiency at the same fuel-per-core setting
-        return 2000000f * fuel * fuel / cores;
+        return 20000f * fuel * fuel / cores; // Delt V - Revert upstream buff for normal AME operation
```
Comment (stellar-novas) [inline Content.Server/Ame/AmeNodeGroup.cs:185]: pleas fix the typo in the comment

File: Resources/Prototypes/DeltaV/Catalog/Fills/Paper/manuals.yml
[the remark this answers is not in the document]
Comment (VMSolidus) [reply Resources/Prototypes/DeltaV/Catalog/Fills/Paper/manuals.yml:1]: No.

[the remark this answers is not in the document]
Comment (DangerRevolution) [reply Resources/Prototypes/DeltaV/Catalog/Fills/Paper/manuals.yml:1]: > Can we localise the names of the lunchboxes? `book-text-lunchbox-title`?

Yes, but later.

[the remark this answers is not in the document]
Comment (DangerRevolution) [reply Resources/Prototypes/DeltaV/Catalog/Fills/Paper/manuals.yml:1]: and in a seperate PR

**Review by DEATHB4DEFEAT — APPROVED** [review]
I have minimally looked at this PR 👍

**Pull request merged**